src/lib/components/pagination/Pagination.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Pagination extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+    this.pageNumbers = this.pageNumbers.bind(this);
+    this.renderPages = this.renderPages.bind(this);
+    this.getPage = this.getPage.bind(this);
+    this.nextPage = this.nextPage.bind(this);
+    this.prevPage = this.prevPage.bind(this);
+  }
+
+  getPage(pageNumber) {
+    this.props.onPageChange(pageNumber);
+  }
+
+  handleClick(event) {
+    event.preventDefault();
+    this.getPage(event.target.id);
+  }
+
+  render() {
+    return (
+      <ul className="pagination">
+        <li onClick={this.prevPage}>{'<'}</li>
+        {this.renderPages()}
+        <li onClick={this.nextPage}>{'>'}</li>
+      </ul>
+    );
+  }
+
+  renderPages() {
```

Comment:
```
  renderPages() {
    const { totalPages } = this.props;
    return [...Array(totalPages).keys()].map(pageNumber => {
      pageNumber = pageNumber + 1 
      return (
        <li id={pageNumber} onClick={this.handleClick} key={pageNumber}>
          {pageNumber}
        </li>
      );
    });
  }
```
And probablt get rid of this.pageNumbers()

Reply:
But not entirely sure if you had more deep thought about  this.pageNumbers(), so your call

Reply:
I didn't have more of a deep thought; just wanted to simplify the function... also I wasn't sure how to make a range quickly in JS :)